frontend/src/app/components/containers/complaints/outcomes/hwcr-equipment/equipment-item.tsx

```diff
@@ -89,171 +88,233 @@ export const EquipmentItem: FC<EquipmentItemProps> = ({ equipment, isEditDisable
+                onClick={() => setShowModal(true)}
               >
-                {equipment.xCoordinate}
-              </div>
+                <i className="bi bi-trash3"></i>
+                <span>Delete</span>
+              </Button>
             </div>
-          </Col>
-        </Row>
-        <Row>
-          <Col
-            xs={12}
-            md={4}
-          >
-            <div className="equipment-item-content">
-              <div className="label">Set by</div>
-              <div className="comp-details-content">
+          </div>
+
+          {/* EQUIPMENT META */}
+          <dl>
+            <div>
+              <dt>Address</dt>
+              <dd>{equipment.address}</dd>
+            </div>
+            <div>
+              <dt>Latitude/Longitude</dt>
+              <dd>
+                {equipment.yCoordinate} {equipment.xCoordinate}
+              </dd>
+            </div>
+            <div>
+              <dt>Set by</dt>
+              <dd>
                 <div
+                  className="comp-avatar comp-avatar-sm comp-avatar-orange"
                   data-initials-sm={getAvatarInitials(`${setEquipmentFullName}`)}
-                  className="comp-pink-avatar-sm"
                 >
-                  <span
-                    id="equipment-officer-set-div"
-                    className="comp-padding-left-xs"
-                  >
-                    {setEquipmentFullName}
-                  </span>
+                  <span id="equipment-officer-set-div">{setEquipmentFullName}</span>
                 </div>
-              </div>
+              </dd>
             </div>
-          </Col>
-          <Col
-            xs={12}
-            md={4}
-          >
-            <div
-              className="equipment-item-content"
-              id="equipment-date-set-div"
-            >
-              <div className="label">Set date</div>
-              <div className="value">{formatDate(setEquipmentDate?.toString())}</div>
+            <div>
+              <dt>Date set</dt>
+              <dd id="equipment-date-set-div">{formatDate(setEquipmentDate?.toString())}</dd>
+            </div>
+            {equipment.id && removedEquipmentActor && (
+              <>
+                <div>
+                  <dt>Removed by</dt>
+                  <dd>
+                    <div
+                      className="comp-avatar comp-avatar-sm comp-avatar-orange"
+                      data-initials-sm={getAvatarInitials(removedEquipmentFullName ?? "")}
+                    >
+                      <span id="comp-details-assigned-officer-name-text-id">{removedEquipmentFullName}</span>
+                    </div>
+                  </dd>
+                </div>
+                <div>
+                  <dt>Removal date</dt>
+                  <dd id="equipment-removal-date">{formatDate(removedEquipmentDate?.toString())}</dd>
+                </div>
+                <div>
+                  <dt>Was animal captured?</dt>
```

Comment:
This should not display when the equipment is type Signage or Trail Camera.   Two ways to get here:
1) Select the equipment as either Signage or Trail Camera - you aren't prompted to enter a value on the form (that's good) however it displays on the read-only screen (that's bad)
2) Edit an equipment from a trap (e.g. Bear Snare) and change it Signage or Trial Camera.   The value is cleared out and but it still says "No" which although is true, is different than how it's currently working in test.

Reply:
Bug fix pushed up to resolve display issue here as requested